flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -618,10 +618,11 @@ TMSlaves() {
 runBashJavaUtilsCmd() {
     local cmd=$1
     local conf_dir=$2
-    local class_path="${3:-$FLINK_BIN_DIR/bash-java-utils.jar:`findFlinkDistJar`}"
+    local class_path=$3
+    local dynamic_args=${@:4}
     class_path=`manglePathList ${class_path}`
 
-    local output=`${JAVA_RUN} -classpath ${class_path} org.apache.flink.runtime.util.BashJavaUtils ${cmd} --configDir ${conf_dir} 2>&1 | tail -n 1000`
+    local output=`${JAVA_RUN} -classpath ${class_path} org.apache.flink.runtime.util.BashJavaUtils ${cmd} --configDir ${conf_dir} $dynamic_args 2>&1 | tail -n 1000`
```

Comment:
To make sure that it works, I suggest to add a test to `BashJavaUtilsITCase` where we rewrite some memory options in `flink-conf.yaml` by dynamic args and check the result.